test/Worker.Extensions.Rpc.Tests/Worker.Extensions.Rpc.Tests.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net7.0;net48</TargetFrameworks>
```

Comment:
Will follow up with @jviau about the removal of net48.
https://azfunc.visualstudio.com/Azure%20Functions/_build/results?buildId=136636&view=logs&j=9a08fc0a-e2b9-596a-b012-19bc88517f7b&t=cad48bfc-f889-5db0-89e3-bef93ff5b948&l=91